### PR TITLE
Physics V2

### PR DESCRIPTION
**Pull request by CedricGuillemet**

### CODE REVIEW

**Comment by RaananW** [comment]
you will need to update the content structure as well, to allow both versions

**Comment by RaananW** [comment]
The file that needs to be updated is the structure.json - to reference the correct files in the right place

**Comment by RaananW** [comment]
Redirects as well, if you want to change th URL of the files

File: configuration/structure.json [hunk 1]
```diff
@@ -1581,45 +1581,97 @@
...
+                                            "children": {},
+                                            "content": "features/featuresDeepDive/physics/v1/advancedPhysicsFeatures"
+                                        },
+                                        "addPhysicsEngine": {
+                                            "friendlyName": "Add Your Own Physics Engine",
+                                            "children": {},
+                                            "content": "features/featuresDeepDive/physics/v1/addPhysicsEngine"
+                                        },
+                                        "content": "features/featuresDeepDive/physics"
+                                    }
```
Comment (RaananW) [inline configuration/structure.json]:
```suggestion
                                },
                                "content": "features/featuresDeepDive/physics"
```

File: configuration/structure.json [hunk 2]
```diff
@@ -1581,45 +1581,97 @@
...
+                                        "advancedPhysicsFeatures": {
+                                            "friendlyName": "Advanced Physics Features",
+                                            "children": {},
+                                            "content": "features/featuresDeepDive/physics/v1/advancedPhysicsFeatures"
+                                        },
+                                        "addPhysicsEngine": {
+                                            "friendlyName": "Add Your Own Physics Engine",
+                                            "children": {},
+                                            "content": "features/featuresDeepDive/physics/v1/addPhysicsEngine"
+                                        },
```
Comment (RaananW) [inline configuration/structure.json]:
```suggestion
                                        }
```

File: content/features/featuresDeepDive/physics/v2/concepts.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+| Enum | Description | Image |
+| --- | --- | --- |
+| SPHERE | Simple sphere | |
+| CAPSULE | A cylinder with a half sphere at top and bottom | |
+| CYLINDER | Cylinder | |
+| BOX | Box | |
+| CONVEX_HULL | A convex hull is the smallest convex shape containing points.| |
+| CONTAINER | Holder of other shapes | |
+| MESH | Mesh used for rendering or a simpler version | |
+| HEIGHTFIELD | A height field mesh is a 2D surface with height data. | |
```
Comment (carolhmj) [inline content/features/featuresDeepDive/physics/v2/concepts.md:37]: We haven't implemented the heightfield yet, so should we take it off the docs?

Comment (RaananW) [reply content/features/featuresDeepDive/physics/v2/concepts.md:37]: The type is supported in the v2 plugin architecture but not in the plugin that was implemented. We can keep it.

**Comment by carolhmj** [comment]
Deprecated in favor of https://github.com/BabylonJS/Documentation/pull/808

**Pull request closed without merging**